modules/vault-cluster/variables.tf

```diff
@@ -176,12 +176,32 @@ variable "enable_s3_backend" {
   default     = false
 }
 
+variable "enable_dynamo_backend" {
+  description = "Whether to configure a DynamoDB storage backend (No Consul)"
+  default     = false
+}
+
 variable "s3_bucket_name" {
   description = "The name of the S3 bucket to create and use as a storage backend. Only used if 'enable_s3_backend' is set to true."
   default     = ""
 }
 
+variable "dynamo_table_name" {
+  description = "The name of the Dynamo Table to create and use as a storage backend. Only used if 'enable_dynamo_backend' is set to true."
+  default     = ""
+}
+
 variable "force_destroy_s3_bucket" {
   description = "If 'configure_s3_backend' is enabled and you set this to true, when you run terraform destroy, this tells Terraform to delete all the objects in the S3 bucket used for backend storage. You should NOT set this to true in production or you risk losing all your data! This property is only here so automated tests of this module can clean up after themselves. Only used if 'enable_s3_backend' is set to true."
   default     = false
 }
+
+variable "dynamo_read_capacity" {
+  description = "Sets the DynamoDB read capacity for storage backend"
+  default     = "5"
+}
+
+variable "dynamo_write_capacity" {
+  description = "Sets the DynamoDB write capacity for storage backend"
+  default     = "5"
```

Comment:
Should these be numbers (i.e., not wrapped in quotes)?

Reply:
They can be but don't have to be.   Changed them for consistency.